packages/terra-demographics-banner/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Changelog
 
 Unreleased
 ----------
+### Updated
```

Comment:
nit: We should keep the CHANGELOG.md sections in alphabetical order, i.e. "Updated" section should come after "Changed".

Reply:
I think changelog should be updated in the order in which changes were made.